packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-ticks.tsx

```diff
@@ -209,6 +242,9 @@ export const AxisTicks = () => {
                             key={`y-grid-tick-${y}`}
                             range={range}
                             tickStep={tickStep[Y]}
+                            // Show the tick labels as multiples of pi
+                            // if the tick step is a multiple of pi.
+                            showPi={tickStep[Y] % Math.PI === 0}
```

Comment:
Ooo this looks great. My brain's a little tired at the end of the day - is there a reason we're not using the `isPiMultiple` function you created here?

Reply:
I didn't use that function because isPiMultiple returns true for some pi fractions. At the moment, I don't think the ticks can handle things like pi/2 or pi/4, so I opted for a direct pi modulo.

I could use the function and add an option to only do integer multiples of pi?

Reply:
Naw, that makes sense now! Thank you for the explanation! :D